docs/streams/core-concepts.html

```diff
@@ -78,7 +78,7 @@ <h3><a id="streams_topology" href="#streams_topology">Stream Processing Topology
     <img class="centered" src="/{{version}}/images/streams-architecture-topology.jpg" style="width:400px">
 
     <p>
-        Kafka Streams offers two ways to define the stream processing topology: the <a href="/{{version}}/documentation/streams/developer-guide#streams_dsl"><b>Kafka Streams DSL</b></a> provides
+        Kafka Streams offers two ways to define the stream processing topology: the <a href="/{{version}}/documentation/streams/developer-guide/dsl-api.html#streams-dsl"><b>Kafka Streams DSL</b></a> provides
```

Comment:
This is for line 88 below: the link to `<a href="#streams_architecture_tasks">` is also broken since we moved that part to `https://kafka.apache.org/11/documentation/streams/architecture#streams_architecture_tasks`, could you fix that as well?

Reply:
Also in line 82 for processor api links.